modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10944,6 +10944,7 @@ move-selected-item-in-x-one-position-up=Move selected item in {0} one position u
 move-selected-items-from-x-to-x=Move selected items from {0} to {1}.
 move-thread=Move Thread
 move-to=Move To
+move-to-current-folder=Move to current folder
```

Comment:
@igor-franca  We need to decide if this is a sentence or a title and adjust the value accordingly.